RST2-Programiranje-Vaje/Vaje_1.cs

```diff
@@ -65,23 +119,78 @@ public static int Naloga123(int mesec)
         /// </summary>
         public static void Naloga131()
         {
-            // Pridobimo trenutni čas
-            DateTime trenutniCas = DateTime.Now;
+            DateTime date1 = DateTime.Now;
+            
+            while((date1.Hour * date1.Minute * date1.Second % 17) != 0) {
+                Thread.Sleep(500);
+                date1 = DateTime.Now;
+            }
+            Console.WriteLine($"{date1} can be divided by 17, when hours, minutes and seconds are multiplied with each other.");
+        }
 
-            // Ponavljamo, dokler produkt ni deljiv s 17
-            while ((trenutniCas.Hour * trenutniCas.Minute * trenutniCas.Second) % 17 != 0)
-            {
-                // Uporabimo formatiranje v interpolaciji
-                Console.WriteLine($"{trenutniCas:HH:mm:ss}");
+        /// <summary>
+        ///Zapišite metodo, ki v danem seznamu realnih števil izbriˇše vse vrednosti,
+        ///ki so manjše od začetnega povprečja. Naredite dve implementaciji: eno z zanko for in eno
+        ///z zanko foreach
+        /// </summary>
+        
+        public static void Naloga132(ref List<float> myArr){
+            float sum = 0;
+            int l = myArr.Count;
+            foreach (float item in myArr) {
+                sum += item;
+            }
+            float avg = sum / l;
 
-                // Po vsakem izpisu počakamo 900 ms
-                Thread.Sleep(900);
-                
-                trenutniCas = DateTime.Now;
+            List<float> tmp = new List<float>();
+
+            foreach(float item in myArr) {
+                if (item > avg ){
+                   tmp.Add(item);
+                }
             }
-            Console.WriteLine($"{trenutniCas:HH:mm:ss} je deljivo s 17.");
+            myArr = tmp;
+        } 
+
+        ///<summary>
+        /// Zapišite metodo, ki z uporabo zanke 
+        /// do izpisuje večkratnike danega naravnega
+        /// števila, dokler vrednost večkratnika ne preseže danega parametra.
+        public static void Naloga133(int k, int m)  {
+            int curr = k;
+            int i = 1;
+            do
+            {   
+                curr = curr * i;
+                Console.WriteLine($"{curr} is smaller than {m}");
+                curr = curr + k;
```

Comment:
This would be considered a bad practice. Only assign to curr once each iteration